docs/adding_software/opening_pr.md

```diff
@@ -95,3 +96,13 @@ git push koala example_branch
 
    If all goes well, one or more bots :robot: should almost instantly create a comment in your pull request
    with an overview of how it is configured - you will need this information when providing build instructions.
+
+### Rebuilding software
+We typically do not rebuild software, since (strictly speaking) this breaks reproducibility for anyone using the software. However, there are certain situations in which it is difficult or impossible to avoid.
+
+To do a rebuild, you add the software you want to rebuild to a dedicated easystack file in the `rebuilds` directory. Include the current date (YYYYMMDD) in the name of the EasyStack file. E.g.:
+```shell
+echo '  - example-1.2.3-GCC-12.3.0.eb' >> easystacks/software.eessi.io/2023.06/rebuilds/20240327-eessi-2023.06-eb-4.9.0-2023a.yml
```

Comment:
I've used a naming scheme `YYYYMMDD-eb-<EB VERSION>-<APP NAME + VERSION>-<SHORT DESCRIPTION>.yml` for the OpenMPI rebuild: `20240301-eb-4.9.0-OpenMPI-4.1.x-fix-smcuda.yml`, shall we stick to that?

Reply:
Now explicitely including this format in the instructions.